Plugins/Flow.Launcher.Plugin.BrowserBookmark/Commands/BookmarkLoader.cs

```diff
@@ -1,20 +1,21 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
-using Flow.Launcher.Infrastructure;
 using Flow.Launcher.Plugin.BrowserBookmark.Models;
 using Flow.Launcher.Plugin.SharedModels;
 
 namespace Flow.Launcher.Plugin.BrowserBookmark.Commands
 {
     internal static class BookmarkLoader
     {
+        public static IPublicAPI publicAPI { get; internal set; }
+
         internal static MatchResult MatchProgram(Bookmark bookmark, string queryString)
         {
-            var match = StringMatcher.FuzzySearch(queryString, bookmark.Name);
+            var match = publicAPI.FuzzySearch(queryString, bookmark.Name);
```

Comment:
are you sure using this here is going to work?

Reply:
because i just tested with similar code, it throw a null reference exception

Reply:
I guess I forget to assign the api🤣. Will check later. These few days are busy...🥲